nucypher/characters/control/specifications/fields/__init__.py

```diff
@@ -0,0 +1,9 @@
+from marshmallow import fields
+from .key import *
+from .treasuremap import *
+from .messagekit import *
+from .datetime import *
+from .label import *
```

Comment:
Should we just add all field type definitions to the same `fields.py` file instead of one per file? Would that save us from needing this code?